src/smt/qi_queue.cpp

```diff
@@ -202,11 +202,26 @@ namespace smt {
         ent.m_instantiated = true;
 
         TRACE("qi_queue_profile", tout << q->get_qid() << ", gen: " << generation << " " << *f << " cost: " << ent.m_cost << "\n";);
+        //NEVER remove coming_from_quant
```

Comment:
Add space between // and first character

Reply:
done